web-registry/src/pages/BatchDetails.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { Box } from '@mui/material';
+import { useParams } from 'react-router-dom';
+
+import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+import Section from 'web-components/lib/components/section';
+import Title from 'web-components/lib/components/title';
+
+import { getBatchWithSupplyForDenom } from '../lib/ecocredit';
+import { getMetadata } from '../lib/metadata-graph';
+import { useBatchDetailsQuery } from '../generated/graphql';
+
+import type { BatchInfoWithSupply } from '../types/ledger/ecocredit';
+import type { BatchMetadataLD } from '../generated/json-ld';
+import {
+  BatchInfoGrid,
+  BatchMetadata,
+  BatchTotalsGrid,
+} from '../components/molecules';
+
+export const BatchDetails: React.FC = () => {
+  const { batchDenom } = useParams();
+  const [batch, setBatch] = useState<BatchInfoWithSupply>();
+  const [metadata, setMetadata] = useState<BatchMetadataLD>();
+
+  useEffect(() => {
+    const fetch = async (): Promise<void> => {
+      if (batchDenom) {
+        try {
+          const batch = await getBatchWithSupplyForDenom(batchDenom);
+          setBatch(batch);
+          if (batch.metadata) {
+            const { data: metadata } = await getMetadata(batch.metadata);
```

Comment:
```suggestion
            const { metadata } = await getMetadata(batch.metadata);
```

I've tested this with https://deploy-preview-866--regen-registry.netlify.app/credit-batches/C01-20170101-20180101-006, a credit batch that I've just created with some metadata but somehow the metadata is not getting displayed while it's indeed returned as you can see on:
![image](https://user-images.githubusercontent.com/3149358/160781747-412e6ba6-b5a3-4623-afdb-d4cd9204b638.png)
The code suggestion above should fix it.